internal/keppel/gc_policy.go

```diff
@@ -245,6 +214,8 @@ type GCStatus struct {
 	// If the image is not protected, contains all policies with action "delete"
 	// that could delete this image in the future.
 	RelevantPolicies []GCPolicy `json:"relevant_policies,omitempty"`
+	// If the image is protected, contains all tag policies that protect this image.
+	ProtectedByTagPolicies []TagPolicy `json:"protected_by_tag_policies,omitempty"`
```

Comment:
This should only show the one tag policy that actually blocked the deletion:

```suggestion
	ProtectedByTagPolicy Option[TagPolicy] `json:"protected_by_tag_policy,omitempty"`
```